File: PlatziCodingChallenge_Dia56/Slides.js
// Slides en JavaScript

function middleChars(text) {
  let length = text.length
  let position = length % 2 === 0 ? length / 2 - 1 : (length - 1) / 2
  let number = length % 2 === 0 ? 2 : 1
  return text.substr(position, number)
}

console.log(middleChars('Platzi')) // at
console.log(middleChars('Colombia')) // om
console.log(middleChars('Brazo')) // a

File: Dia56/Slides.py
# Slides en Python

def middleChars(text):
  length = len(text)
  if length % 2 == 0:
    position = round(length / 2 - 1)
    return text[position] + text[position + 1]
  else:
    position = round((length - 1) / 2)
    return text[position]

print(middleChars('Platzi')) # at
print(middleChars('Colombia')) # om
print(middleChars('Brazo')) # a